pkg/config/config.go

```diff
@@ -207,6 +207,8 @@ func (c *Config) GetStepConfig(flagValues map[string]interface{}, paramJSON stri
 
 		stepConfig.mixInHookConfig(def.Hooks)
 	}
+	// read hook info from user config file (.pipeline/config.yaml), overwrites hook config from defaults
+	stepConfig.mixInHookConfig(c.Hooks)
```

Comment:
Will this override the whole config? If a user has e.g. for ANS a hook config provided will the Splunk and sentry hook still be active (if read from defaults)?

Reply:
It should not, I will provide a unit test to check this!

Reply:
[Tests have been added, it works as expected. ](https://github.com/SAP/jenkins-library/pull/3675/files#diff-29cea1b5b831c8655c7155f43e2367cec73e66d1e338f8b3c7877a2f339b8811R939)

Reply:
@ffeldmann This will allow the user to overwrite the default Splunk and Sentry hook configuration with their config.yaml. Is that still ok with you?